activemq/datadog_checks/activemq/data/conf.yaml.example

```diff
@@ -1,35 +1,95 @@
-##########
-# WARNING
-##########
-# This sample works only for ActiveMQ >= 5.8.0
-# If you are running a version older than that, you can refer to agent 5.10.x released
-# sample files, https://raw.githubusercontent.com/DataDog/dd-agent/5.10.1/conf.d/activemq.yaml.example
+## WARNING
+## This sample works only for ActiveMQ >= 5.8.0
+## If you are running a version older than that, you can refer to agent 5.10.x released
+## sample files, https://raw.githubusercontent.com/DataDog/dd-agent/5.10.1/conf.d/activemq.yaml.example
 
 instances:
+  
+  ## @param host - string - required
+  ## 
+  #
   - host: localhost
+
+  ## @param port - integer - required
+  ##
+  #
     port: 1099
-  #   user: username
-  #   password: password
-  #   process_name_regex: .*process_name.* # Instead of specifying a host, and port. The agent can connect using the attach api.
-  #                                        # This requires the JDK to be installed and the path to tools.jar to be set below.
-  #   tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar # To be set when process_name_regex is set
+
+  ## @param user - string - optional
+  ##
+  #
+  #  user: <USERNAME>
+  
+  ## @param password - string - optional
+  ## XXXXXXXXXXX
+  #
+  #  password: <PASSWORD>
+
+  ## @param process_name_regex - string - optional
+  ## XXXXXXXXXXX
+  #
+  #  process_name_regex: .*process_name.* 
+
+  ## @param tools_jar_path - string - optional
+  ## Instead of specifying a host, and port. The agent can connect using the attach api.
+  ## This requires the JDK to be installed and the path to tools.jar to be set below.
```

Comment:
Looks like these two lines are actually the description for `process_name_regex`